clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -74,6 +75,32 @@ class LLVMLoweringInfo {
 class CIR_Op<string mnemonic, list<Trait> traits = []> :
     Op<CIR_Dialect, mnemonic, traits>, LLVMLoweringInfo;
 
+//===----------------------------------------------------------------------===//
+// GlobalOp
+//===----------------------------------------------------------------------===//
+
+// TODO(CIR): For starters, cir.global has only name and type.  The other
+// properties of a global variable will be added over time as more of ClangIR
+// is upstreamed.
+
+def GlobalOp : CIR_Op<"global"> {
+  let summary = "Declare or define a global variable";
+  let description = [{
+    ... lots of text to be added later ...
```

Comment:
Would still prefer a bit of a stub here, even if it is incomplete (rather than just a 'todo') as this makes it to our documentation.